cds_ils/importer/providers/safari/ignore_fields.py

```diff
@@ -177,12 +177,15 @@
     "300__b",
     "300__c",
     "306__a",
+    "336__0",
```

Comment:
All changes in this file are closing https://github.com/CERNDocumentServer/cds-ils/issues/977